src/SBarcodeGenerator.h

```diff
@@ -41,7 +41,7 @@ class SBarcodeGenerator : public QQuickItem
      * \fn ~SBarcodeGenerator() override
      * \brief Destructor.
      */
-    ~SBarcodeGenerator() override { };
+    ~SBarcodeGenerator() override { }
```

Comment:
This destructor is actually not needed AFAIK :)

Reply:
Deleted